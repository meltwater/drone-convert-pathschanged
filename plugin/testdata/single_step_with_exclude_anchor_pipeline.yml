kind: pipeline
type: docker
name: default

commands: &commands
  commands:
  - echo "This step will be excluded when README.md is changed"

steps:
- <<: *commands
  name: message
  image: busybox
  when:
    paths:
      exclude:
      - README.md
