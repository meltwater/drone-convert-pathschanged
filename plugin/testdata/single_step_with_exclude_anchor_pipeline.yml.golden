kind: pipeline
type: docker
steps:
- when:
    paths:
      exclude:
      - README.md
    event:
      exclude:
      - '*'
  commands:
  - echo "This step will be excluded when README.md is changed"
  image: busybox
  name: message
commands:
  commands:
  - echo "This step will be excluded when README.md is changed"
name: default
